openhub/portfolio_projects.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from community.git import get_owner
+from openhub.models import PortfolioProject, PortfolioProjectActivity
+
+
+def get_portfolio_projects_data(json_object):
+    data = json_object['response']['result'
+                                   ]['portfolio_projects']['project']
+    return data
+
+
+def import_data(project):
+    name = project.get('name', None)
+    activity = project.get('activity', None)
+    i_use_this = project.get('i_use_this', None)
+    community_rating = project.get('community_rating', None)
+    org = get_owner()
+
+    try:
+        (cr, create) = PortfolioProjectActivity.objects.get_or_create(
+            **project['twelve_mo_activity_and_year_on_year_change']
+            )
+        if create:
+            cr.save()
+        (c, created) = PortfolioProject.objects.get_or_create(
+            twelve_mo_activity_and_year_on_year_change=cr,
+            name=name,
```

Comment:
`**`

Reply:
I have used `**` everywhere and its working fine but don't why that is not working in this case, I have tried every possible way.